package.json

```diff
@@ -10,6 +10,24 @@
   "workspaces": [
     "packages/**"
   ],
+  "release": {
+    "branches": [
+      "master"
+    ],
+    "plugins": [
+      "@semantic-release/commit-analyzer",
+      "@semantic-release/release-notes-generator",
+      "@semantic-release/changelog",
+      [
+        "@semantic-release/git",
+        {
+          "assets": [
+            "docs"
+          ]
+        }
+      ]
+    ]
+  },
```

Comment:
I think in this current configuration, `semantic-release` will by default attempt to publish the NPM package found at the root, a.k.a. [this package](https://github.com/blockstack/stacks.js/blob/master/package.json#L2). If you intend to have that package published on release, you'll need to add more items to the `assets` block to ensure updated files get pushed back to the repo properly after `semantic-release` runs on the GH runner. [See example here.](https://github.com/hirosystems/gh-actions-example/blob/main/package.json#L60-L71)

If you _don't_ want this root package to be published to npmjs.org, you'll need to [add an extra configuration like so](https://github.com/hirosystems/gh-actions-example/blob/main/package.json#L48-L54).

Also note, you currently have `docs` listed as an asset to be pushed back up to the `master` branch upon a release. The `docs` dir is not listed as an [ignored path in the workflow file](https://github.com/blockstack/stacks.js/pull/1062/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R9), so pushing changes to this dir on a release will cause another workflow to be ran. Probably harmless, but not sure if that's what you intended. At most it would just be unnecessary to run the workflow again.

Reply:
- Added config auto publish false for npm . I would suggest to do it in separate PR once we have detailed requirements as per your questions asked in this thread 
- docs path is ignored